cpu/pmqa.py

```diff
@@ -20,9 +20,12 @@
 
 from avocado import Test
 from avocado import main
-from avocado.utils import process, git
+from avocado import skipIf
+from avocado.utils import process, git, cpu
 from avocado.utils.software_manager import SoftwareManager
 
+IS_POWER_NV = 'PowerNV' in cpu._get_cpu_info()
```

Comment:
I don't think so it working any time as `_get_cpu_info()` only pares first entry as in my system below is the output, can you pleas help me to understand more 

>>> cpu._get_cpu_info()
[b'processor\t: 0\n', b'cpu\t\t: POWER9, altivec supported\n', b'clock\t\t: 2700.000000MHz\n', b'revision\t: 2.2 (pvr 004e 1202)\n']
>>>